src/coreclr/vm/threadsuspend.cpp

```diff
@@ -4545,6 +4545,30 @@ struct ExecutionState
     ExecutionState() : m_FirstPass(TRUE) {LIMITED_METHOD_CONTRACT;  }
 };
 
+#if defined(TARGET_ARM64)
+static inline void* PacSignPtr(void* ptr)
+{
+    __asm__ volatile (".arch_extension pauth\n"
```

Comment:
i guess to compile this, you need to pass the flags that you intended to pass in https://github.com/dotnet/runtime/pull/108561, right?

Reply:
On Linux, it shouldn't require compile time flag as we are specifying that path is required in assembly itself. This would need to be refactored when we will merge #108561. However, I'm not sure yet about the changes to inline assembly that we need to make builds happy on Windows.